src/smart/SmartPatient.js

```diff
@@ -101,8 +101,8 @@ export function SmartPatient() {
         console.log(e);
       }
 
-      let epTypeStr = process.env?.REACT_APP_CCSM_EPISODEOFCARE_TYPES ?? 'urn:oid:1.2.840.114350.1.13.284.2.7.4.726668.130|2,urn:oid:1.2.840.114350.1.13.284.3.7.4.726668.130|2';
-      
+      let epTypeStr = process.env?.REACT_APP_CCSM_EPISODEOFCARE_TYPES ?? 'urn:oid:1.2.840.114350.1.13.284.3.7.4.726668.130|2';
```

Comment:
Probably not a huge deal since UMMC will be setting the environment variable, but I wonder if we should default `epTypeStr` to the PRD value as opposed to the nonPRD value, since they will primarily be using the PRD environment going forward.